backend/tests/integration/tools/test_views.py

```diff
@@ -0,0 +1,125 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+import pytest
+from freezegun import freeze_time
+from helpers.utils import create_record
+from werkzeug.datastructures import FileStorage
+
+from inspirehep.tools.utils import find_references
+
+
+@pytest.fixture(scope="function")
```

Comment:
```suggestion
@pytest.fixture(scope="function")
@freeze_time("2020-06-12")
```

intead of ``with`` :)

Reply:
It doesn't work if I use both decorators, that's why I had to use the `with`